internal/injector/aspect/advice/call.go

```diff
@@ -120,6 +121,53 @@ func (a *appendArgs) AddedImports() []string {
 	return imports
 }
 
+type replaceFunction struct {
+	path string
+	name string
+}
+
+// ReplaceFunction replaces the called function with the provided drop-in replacement. The signature
+// must be compatible with the original function (it may accept a new variadic argument).
```

Comment:
Is there a way to check if it truely is compatible ? Maybe not at runtime but when generating go code from yaml

Reply:
The issue is that'd need us to correlate the advice with the join-point, which would be non-trivial...